index.html

```diff
@@ -1,51 +1,236 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <script
-      src="https://kit.fontawesome.com/a70cf7601d.js"
-      crossorigin="anonymous"
-    ></script>
-    <link rel="stylesheet" href="./css/styles.css" />
-    <title>Personal Portfolio</title>
-  </head>
-
-  <body>
-    <div>
-      <header class="header">
-        <nav class="nav">
-          <a href="">Welcome</a>
-          <i class="fas fa-bars"></i>
-        </nav>
-      </header>
-      <section class="hero-heading">
-        <h1 class="title">Hey there, I am Dennis.</h1>
-        <p id="title">I'm a software Developer</p>
-        <p class="description">
-          I can help you build a product, feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
-        <ul class="social-icons">
-          <li>
-            <a href=""><img src="./images/icons/gitHub.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./images/icons/linkedin.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./images/icons/angelist.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./images/icons/twitter.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./images/icons/medium.png" alt="" /></a>
-          </li>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <script src="https://kit.fontawesome.com/a70cf7601d.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="./css/styles.css" />
+  <title>Personal Portfolio</title>
+</head>
+
+<body>
+  <div>
```

Comment:
- [ ] I will suggest you remove this top `div`. Its not part of the requirements. Our aim is to use semantic HTML for best practices. So adding it defeats the end goal. Check this [page](https://www.w3schools.com/html/html5_semantic_elements.asp) for more information

Reply:
ok resolved. Thank you.